client/my-sites/backup/backup-date-picker/hooks.ts

```diff
@@ -62,7 +51,7 @@ export const useCanGoToDate: CanGoToDateHook = (
 			// then everything's fine (this should never happen)
 			return true;
 		},
-		[ selectedDateIsVisible, selectedDate, today, oldestDateAvailable, hasNoBackups ]
+		[ true, selectedDate, today, oldestDateAvailable, hasNoBackups ]
```

Comment:
Is `true` needed on this dependency array for `useCallback`? Given that we removed the `selectedDateIsVisible` logic inside this hook, we should only re-create it if the other dependencies change (`selectedDate`, `today`, `oldestDateAvailable`, `hasNoBackups`).